src/mumble/CoreAudio.cpp

```diff
@@ -84,33 +84,46 @@ const QHash<QString, QString> CoreAudioSystem::getDevices(bool input) {
 	UInt32 len, ndevs;
 	OSStatus err;
 
-	err = AudioHardwareGetPropertyInfo(kAudioHardwarePropertyDevices, &len, NULL);
+	AudioObjectPropertyAddress propertyAddress = {
+			kAudioHardwarePropertyDevices,
+			kAudioObjectPropertyScopeGlobal,
+			kAudioObjectPropertyElementMaster
+	};
+
+	err = AudioObjectGetPropertyDataSize(kAudioObjectSystemObject, &propertyAddress, 0, nullptr, &len);
 	if (err != noErr)
 		return qhReturn;
 
 	ndevs = len / sizeof(AudioDeviceID);
 	AudioDeviceID devs[ndevs];
 
-	err = AudioHardwareGetProperty(kAudioHardwarePropertyDevices, &len, devs);
+	err = AudioObjectGetPropertyData(kAudioObjectSystemObject, &propertyAddress, 0, nullptr, &len, devs);
 	if (err != noErr)
 		return qhReturn;
 
+	if (input) {
+		propertyAddress.mScope = kAudioDevicePropertyScopeInput;
+	} else {
+		propertyAddress.mScope = kAudioDevicePropertyScopeOutput;
+	}
```

Comment:
Normally I'm not a big fan of 1-liners but in the case of a conditional assignment, I think they are actually more readable:
```suggestion
	propertyAddress.mScope = input ? kAudioDevicePropertyScopeInput : kAudioDevicePropertyScopeOutput;
```